From 86b9d98d2bfb13174df375ff525a7d456a89f729 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Tue, 13 Oct 2015 16:50:25 +0800
Subject: [PATCH 0889/1691] MGS-1132 ARM: Kconfig: increase FORCE_MAX_ZONEORDER
 for ARCH_MXC

commit 86b9d98d2bfb13174df375ff525a7d456a89f729 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Need increase the FORCE_MAX_ZONEORDER to 14 for high resolution camera
(GPU 2D user case). The default value 11(4MB) is not enough now.

Signed-off-by: Jason Liu <r64343@freescale.com>
(cherry picked from commit fff642ffe868cb55f5caec0501e36fd28b6ece50)
---
 arch/arm/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 19f4cc6..cf92325 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1689,6 +1689,7 @@ config FORCE_MAX_ZONEORDER
 	range 11 64 if ARCH_SHMOBILE_LEGACY
 	default "12" if SOC_AM33XX
 	default "9" if SA1111 || ARCH_EFM32
+	default "14" if ARCH_MXC
 	default "11"
 	help
 	  The kernel memory allocator divides physically contiguous memory
-- 
1.9.1

